biz.aQute.bndlib.tests/test/aQute/bnd/osgi/ClazzTest.java

```diff
@@ -0,0 +1,32 @@
+package aQute.bnd.osgi;
```

Comment:
There is already a `test.ClazzTest` test class in this project. I don't think we want a second class with the same simple name in the project. Since there are already tests in the project for method parameters. I don't think we need another test class for this. If you want to keep this test method, please relocate it to the existing `test.ClazzTest` class.

Reply:
i relocated the method.